velox/dwio/common/ColumnSelector.cpp

```diff
@@ -318,15 +315,15 @@ const FilterTypePtr& ColumnSelector::process(const std::string& column, bool) {
 std::pair<std::string_view, std::string_view> extractColumnName(
     const std::string_view& name) {
   // right now this is the only supported expression for MAP key filter
-  auto pos = name.find('#');
-  if (pos != std::string::npos) {
-    // special map column handling
-    auto colName = name.substr(0, pos);
-    auto expr = name.substr(pos + 1);
-    return std::make_pair(colName, expr);
+  const auto pos = name.find('#');
+  if (pos == std::string::npos) {
+    return std::make_pair(name, "");
   }
 
-  return std::make_pair(name, "");
+  // special map column handling
```

Comment:
nit: seems we keep consistent about the comment starting w/ `///`.

Reply:
/// is for public comments in header file. // for private comment or implementation comments.